rootdir/vendor/etc/init/hw/init.common.rc

```diff
@@ -112,6 +112,8 @@ on late-fs
     # cause services for bootanim not running.
     wait_for_prop hwservicemanager.ready true
 
+    mount none /vendor/oem /oem bind rec
```

Comment:
Can you split this into a separate file please?

Reply:
Can we do that later? It's a lil late, the implementation has to go live for release....